### ISSUE TITLE
devexp: Default to thrift if and only if the endpoint contains "::"

### ISSUE THREAD

**Issue by Raynos** [uber/tcurl#56]
All thrift endpoints contain "::"

If none of the the `--thrift`, `--json` or `--raw` flags are set we should default to thrift if "::" and default to json otherwise.

We can re-consider thrift vs json as the global default in a separate issue.

cc @malandrew @kriskowal @ShanniLi @eculver

**Comment by kriskowal** [comment]
Good idea, especially if that implies that tcurl should hit the Meta::thrift* endpoint first to obtain the IDL unless it was already supplied from one on hand. We could simply infer whether as=json or as=thrift from this heuristic and not have that separate discussion about a global default.

**Comment by kriskowal** [comment]
This has landed in #60 but is moot until we get #61 (obtain IDL from Meta::thriftIDL) which is blocking on a proper implementation of Meta::thriftIDL with multiple sources.
